### DISCUSSION TITLE
Mimic hour precision

### DISCUSSION THREAD

**Discussion by MFosset** [MIT-LCP/mimic-code#1375] [MIMIC-IV] [answered]
Hello,

I'd like to know if there is a shift on hour of events added to the shift on dates, because we'd like to work on events that happens at night VS during day in ICU.
For example, an arterial systolic blood pressure recorded at 09:36 am is really in the morning ? 

Thank you

**Comment by marymlucas** [comment, marked as answer]
@MFosset in the release paper it says the offset is measured in days, so I assume the hours would remain consistent. 

_"Finally, date and times were shifted randomly into the future using an offset measured in days. A single date shift was assigned to each subject_id. "_
[https://physionet.org/content/mimiciv/2.0/](https://physionet.org/content/mimiciv/2.0/)

That's just my interpretation of it.

Mary

**Reply by tompollard** [reply]
Thanks Mary! That's correct, time of day is retained.

**Reply by MFosset** [reply]
Thank you to both of you !
